.github/workflows/build.yml

```diff
@@ -30,5 +31,5 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
       - run: python -m pip install --upgrade pip setuptools wheel
-      - run: python -m pip install --upgrade --editable .
-      - run: python setup.py test
+      - run: python -m pip install --upgrade --editable . -r requirements-dev.txt
+      - run: python -m pytest --verbose ./httpie ./tests
```

Comment:
FTR `--doctest-modules` is missing here because it is already defined in `setup.cfg`.